makefile

```diff
@@ -35,6 +43,37 @@ release: | tmp/output/release/
 	@test $(VERSION)
 	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
 
+# Mappings
+# todo: I'd like to do USE_PREBUILT=false, but this error is occuring:
+#  cp: cannot stat 'tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv': No such file or directory
+#  ...even though I checked and the file at that path does exist.
+USE_PREBUILT=true
+
+tmp/input/mondo/:
+	rm -rf $@ &&\
+	cd tmp/input/ &&\
+	git clone --depth 1 https://github.com/monarch-initiative/mondo
+
+tmp/input/mondo.sssom.tsv: tmp/input/mondo/
```

Comment:
## Goal for `mondo.sssom.tsv`
### FYI: `make` goal differences from `mondo-ingest` approach
I deviated a bit from what was in `mondo-ingest`. What I have here makes more sense to me (in general & for this specific use case) compared to how `mondo-ingest` does it:
```make
USE_MONDO_RELEASE=false

tmp/mondo_repo_built:
	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
		rm -rf ./mondo/ &&\
		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
		cd mondo/src/ontology &&\
		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
		cd ../../../../ &&\
		touch $@; fi

$(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@

```

### `USE_PREBUILT=true`
I wanted to get the latest `mondo.sssom.tsv` from the `mondo` repo by cloning the repo and running `make mondo.owl mappings`, but I got an error:
`cp: cannot stat 'tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv': No such file or directory`

Not sure why the error; I checked and the file at that path does exist.

For now, I just set this to `true`, and it simply downloads the `mondo.sssom.tsv` from the latest commit in `mondo`.